src/tools/kata-ctl/src/arch/aarch64/mod.rs

```diff
@@ -27,11 +33,11 @@ mod arch_specific {
     }];
 
     pub fn check(_args: &str) -> Result<()> {
-        println!("INFO: check: aarch64");
+        info!(sl!(), "check: aarch64");
         if Path::new(KVM_DEV).exists() {
             println!("Kata Containers can run on this host\n");
```

Comment:
One question we had - we initially elected not to port over every single println!() call into the logging framework. We only included lines that had WARNING or INFO, etc. in their text. 

Would you like us to port over the rest of these as well? Basically - converting all println!() calls?

Reply:
@byron-marohn IMO, if you already ported some of a similar type, you should probably cover all of them. In this case I would use INFO here

Reply:
Done